### PR TITLE
Use the requests library if it's installed

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

File: bugsnag/notification.py [hunk 1]
```diff
@@ -20,27 +20,49 @@
...
+            if proxy_host:
+                proxy = {
+                    'https': proxy_host,
+                    'http': proxy_host
+                }
+                r = requests.post(url, data=payload, proxies=proxy)
+            else:
+                r = requests.post(url, data=payload)
+
+            status = r.status_code;
```
Comment (kylef) [inline bugsnag/notification.py]: Semi-colon in Python? 😬

File: bugsnag/notification.py [hunk 2]
```diff
@@ -20,27 +20,49 @@
...
+
+            if status != 200:
```
Comment (kylef) [inline bugsnag/notification.py]: I'd change the hard-coded 200 value to `requests.codes.ok`

File: bugsnag/notification.py (hunk 1 above)
Comment (GrahamCampbell) [inline bugsnag/notification.py]: haha, thanks!

File: bugsnag/notification.py (hunk 2 above)
Comment (GrahamCampbell) [inline bugsnag/notification.py]: Ok, sure. :)

**Comment by gkze** [comment]
LGTM. Maybe add tests? CC @kattrali

**Comment by GrahamCampbell** [comment]
🚀

**Pull request merged**